docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -844,6 +844,81 @@ The default tenant configuration is automatically disabled when `quarkus.oidc.au
 
 Note that tenant specific configurations can also be disabled, for example: `quarkus.oidc.tenant-a.tenant-enabled=false`.
 
+== Authentication outside of the HTTP request context
+
+Sometimes it is practical to be able to create `SecurityIdentity` for given token when there is no active request context.
+The `quarkus-oidc` extension provides `io.quarkus.oidc.TenantIdentityProvider` to convert a token to a `SecurityIdentity` instance.
+When the provider is used during an HTTP request, the tenant configuration can be resolved as described in the <<static-tenant-resolution>> section.
+However, when there is no active HTTP request, you need to select tenant explicitly with the `io.quarkus.oidc.TenantFeature` qualifier.
+One situation when you need to select the tenant configuration this way is when you are processing messages with the xref:vertx.adoc#event-bus[Vert.x event bus].
```

Comment:
```suggestion
For example, one situation when you need to verify the token after HTTP request has completed is when you are processing messages with the xref:vertx.adoc#event-bus[Vert.x event bus].
```

Reply:
Suggesting not to focus the attention on the tenant configuration but on the main feature